tests/components/plugwise/test_config_flow.py

```diff
@@ -84,8 +143,93 @@ async def test_form_cannot_connect(hass, mock_smile):
 
     result2 = await hass.config_entries.flow.async_configure(
         result["flow_id"],
-        {"host": "1.1.1.1", "password": "test-password"},
+        {"host": TEST_HOST, "password": TEST_PASSWORD},
     )
 
-    assert result2["type"] == "form"
+    assert result2["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result2["errors"] == {"base": "cannot_connect"}
+
+
+async def test_form_other_problem(hass, mock_smile):
+    """Test we handle cannot connect error."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+
+    mock_smile.connect.side_effect = TimeoutError
+    mock_smile.gateway_id = "0a636a4fc1704ab4a24e4f7e37fb187a"
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {"host": TEST_HOST, "password": TEST_PASSWORD},
+    )
+
+    assert result2["type"] == data_entry_flow.RESULT_TYPE_FORM
+    assert result2["errors"] == {"base": "unknown"}
+
+
+async def test_show_zeroconf_form(hass, mock_smile) -> None:
+    """Test that the zeroconf confirmation form is served."""
+    flow = config_flow.PlugwiseConfigFlow()
```

Comment:
Always start the tests with the core interfaces. Here use `hass.config_entries.flow.async_init`.

Reply:
Overlooked when refitting between PRs, that `def` is redundant now that 78: `async def test_zeroconf_form(hass):` is in. Update PR will be in later today